models/LinearSwitcher/ls.jl

```diff
@@ -0,0 +1,106 @@
+    X = HPolyhedron([HalfSpace([-1.0, 0.0, 0.0, 0.0, 0.0], -3.0 + ε)]) # x >= 3
+    q1 = ConstrainedLinearControlContinuousSystem(A, B, X, U)
+
+    id_location = 2
+    A, B, c = get_coeffs(model["flows"][id_location], n, m, state_vars, input_vars)
+    X = HPolyhedron([HalfSpace([-1.0, 0.0, 0.0, 0.0, 0.0], -2.0 + ε)]) # x >= 2
+    q2 = ConstrainedLinearControlContinuousSystem(A, B, X, U)
+
+    id_location = 3
+    A, B, c = get_coeffs(model["flows"][id_location], n, m, state_vars, input_vars)
+    X = HPolyhedron([HalfSpace([-1.0, 0.0, 0.0, 0.0, 0.0], -1.0 + ε)]) # x >= 1
+    q3 = ConstrainedLinearControlContinuousSystem(A, B, X, U)
+
+    id_location = 4
+    A, B, c = get_coeffs(model["flows"][id_location], n, m, state_vars, input_vars) # new function in SX
+    X = HPolyhedron([HalfSpace([-1.0, 0.0, 0.0, 0.0, 0.0], 0.0 + ε)]) # x >= 0
+    q4 = ConstrainedLinearControlContinuousSystem(A, B, X, U)
+
+    id_location = 5
+    A, B, c = get_coeffs(model["flows"][id_location], n, m, state_vars, input_vars) # new function in SX
+    X = HPolyhedron([HalfSpace([1.0, 0.0, 0.0, 0.0, 0.0], 1.0+ε)]) # x <= 1
+    q5 = ConstrainedLinearControlContinuousSystem(A, B, X, U)
+
+    # automaton structure
+    automaton = LightAutomaton(5)
+
+    modes = [q1, q2, q3, q4, q5]
+
+    # transitions
+    add_transition!(automaton, 1, 2, 1)
+    add_transition!(automaton, 2, 3, 2)
+    add_transition!(automaton, 3, 4, 3)
+    add_transition!(automaton, 4, 5, 4)
+    add_transition!(automaton, 5, 1, 5)
+
+    # guards
+    G12 = HPolyhedron([HalfSpace([1.0, 0.0, 0.0, 0.0, 0.0], 3.0 + ε),
+                       HalfSpace([-1.0, 0.0, 0.0, 0.0, 0.0], -3.0 + ε)]) # x1 = 3
+
+    G23 = HPolyhedron([HalfSpace([1.0, 0.0, 0.0, 0.0, 0.0], 2.0 + ε),
+                       HalfSpace([-1.0, 0.0, 0.0, 0.0, 0.0], -2.0 + ε)]) # x1 = 2
+
+    G34 = HPolyhedron([HalfSpace([1.0, 0.0, 0.0, 0.0, 0.0], 1.0 + ε),
+                       HalfSpace([-1.0, 0.0, 0.0, 0.0, 0.0], -1.0 + ε)]) # x1 = 1
+
+    G45 = HPolyhedron([HalfSpace([1.0, 0.0, 0.0, 0.0, 0.0], 0.0 + ε),
+                       HalfSpace([-1.0, 0.0, 0.0, 0.0, 0.0], -0.0 + ε)]) # x1 = 0
+
+    G51 = HPolyhedron([HalfSpace([1.0, 0.0, 0.0, 0.0, 0.0], 1.0 + ε),
+                       HalfSpace([-1.0, 0.0, 0.0, 0.0, 0.0], -1.0 + ε)]) # x1 = 1
+
+    resetmaps = [ConstrainedIdentityMap(2, G12),
+                 ConstrainedIdentityMap(2, G23),
+                 ConstrainedIdentityMap(2, G34),
+                 ConstrainedIdentityMap(2, G45),
+                 ConstrainedIdentityMap(2, G51)];
+
+    # switching
+    switchings = [AutonomousSwitching()]
+
+    ℋ = HybridSystem(automaton, modes, resetmaps, switchings)
+
+    # initial condition in mode q1
+    initial_condition = [(1, X0)]
+
+    problem = InitialValueProblem(ℋ, initial_condition)
+
+    border = HPolyhedron([HalfSpace([1.0; zeros(4)], -1.2)]) # x1 <= -1.2
```

Comment:
```suggestion
    border = HalfSpace([1.0, 0.0, 0.0, 0.0, 0.0], -1.2)  # x1 <= -1.2
```
(to be consistent with the other vectors)